images/macos/provision/core/chrome.sh

```diff
@@ -14,7 +14,8 @@ echo "Google Chrome version is $FULL_CHROME_VERSION"
 # Get Google Chrome versions information
 CHROME_PLATFORM="mac-$arch"
 CHROME_VERSIONS_URL="https://googlechromelabs.github.io/chrome-for-testing/latest-patch-versions-per-build-with-downloads.json"
-CHROME_VERSIONS_JSON=$(curl -fsSL "${CHROME_VERSIONS_URL}")
+download_with_retries "$CHROME_VERSIONS_URL" "/tmp" "latest-patch-versions-per-build-with-downloads.json"
+CHROME_VERSIONS_JSON=$(cat /tmp/latest-patch-versions-per-build-with-downloads.json)
```

Comment:
What do you think about using either shorter file name or getting it with `basename` command or Shell Parameter Expansion? Also you can omit file name in `download_with_retries` as it's being extracted from url by default.

```suggestion
download_with_retries "$CHROME_VERSIONS_URL" "/tmp"
CHROME_VERSIONS_JSON=$(cat /tmp/${CHROME_VERSIONS_URL##*/})
```

Reply:
I'm fine with original filename, thanks